### PR TITLE
add env for cluster-cidr and service-cidr

### LINKED ISSUE #372: clusternet agent  discoverClusterCIDR  failed
#### What would you like to be added:
image
clusternet-agent failed when  running  in a kubernetes  that apiserver、controller-manager、kube-scheduler run as a docker container or bin
#### Why is this needed:
add param to set cluster cidr、servicecidr ，so clusternet can manage a cluster that apiserver、controller-manager、kube-scheduler run as a docker container or bin，for example  rancher rke cluster /k3s cluster

image

### PR DESCRIPTION
**Pull request by snstaberah**
a easy way to manage rke and k3s，tested with rke and it works well
捕获
image

Signed-off-by: snstaberah <yangyonglc@inspur.com>

#### Which issue(s) this PR fixes:

Fixes #372

### CODE REVIEW

File: pkg/controllers/clusters/clusterstatus/network.go [hunk 1]
```diff
@@ -30,16 +31,24 @@ import (
 // findServiceIPRange returns the cluster IP range for the cluster service.
 // copied from submariner.io/submariner-operator/pkg/discovery/network/generic.go and modified
 func findServiceIPRange(podLister corev1Lister.PodLister) (string, error) {
+
 	clusterIPRange := findPodCommandParameter(podLister, "kube-apiserver", "--service-cluster-ip-range")
 	if clusterIPRange != "" {
 		return clusterIPRange, nil
 	}
+	// Try to find the pod IP range from the env.
+	// fmt.Println("serviceCIDR from env:", os.Getenv("SERVICE_CIDR"))
```
Comment (dixudx) [inline pkg/controllers/clusters/clusterstatus/network.go:40]: remove this line.

File: pkg/controllers/clusters/clusterstatus/network.go [hunk 2]
```diff
@@ -57,7 +66,12 @@ func findPodIPRange(nodeLister corev1Lister.NodeLister, podLister corev1Lister.P
 	if podIPRange != "" {
 		return podIPRange, nil
 	}
-
+	// Try to find the pod IP range from the env.
+	// fmt.Println("clusterCIDR from env:", os.Getenv("CLUSTER_CIDR"))
```
Comment (dixudx) [inline pkg/controllers/clusters/clusterstatus/network.go:70]: remove this line

**Review by dixudx — COMMENTED** [review]
Nice enhancements. Please remove redundant comments. Thanks for the contribution.

**Comment by snstaberah** [comment]
> Nice enhancements. Please remove redundant comments. Thanks for the contribution.

done，please take a look

**Review by dixudx — APPROVED** [review]

**Pull request merged**